src/Core/Content/Product/SalesChannel/Sorting/ProductSortingEntity.php

```diff
@@ -52,6 +52,7 @@ public function createDalSorting(): array
         $sorting = [];
 
         $fields = $this->fields;
+        array_push($fields, $this->getFallbackSortingField());
```

Comment:
```suggestion
        $fields[] = , $this->getFallbackSortingField();
```

Reply:
I accidentally commited that with the , still there :D